code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1382,11 +1382,23 @@
 	var/current_size = 1
 
 /datum/reagent/growthserum/on_mob_life(mob/living/carbon/H)
-	if(volume >= 20 && current_size != 2)
+	if(volume >=200 && current_size != 5)
```

Comment:
make your changes, commit, push to the same branch, the PR will update automatically. (in this case the banch is MisterTikva:patch-2)